src/components/Navbar/Navbar.jsx

```diff
@@ -1,12 +1,21 @@
-import React from 'react';
+import { React } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { HStack, Flex, Button, Image } from '@chakra-ui/react';
+import { HStack, Flex, Button, Image, useToast } from '@chakra-ui/react';
 import NavbarLink from './NavbarLink';
 import ProfileDropdown from './ProfileDropdown';
 import logo from '../../assets/OCHlogo.png';
+import Toast from '../Toast';
 
-const Navbar = ({ isAdmin }) => {
+const Navbar = ({ isAdmin, changesMade }) => {
+  const toast = useToast();
+  // TO DO: get profile image and name
+  const user = {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'user' is assigned a value but never used.